react/src/views/Box/BoxView.tsx

```diff
@@ -294,12 +309,22 @@ function BTBox() {
           numberOfItems: (boxData?.numberOfItems || 0) - (boxFormValues?.numberOfItems || 0),
         },
       })
-        .then(() => {
+        .then((res) => {
+          notificationVar({
+            title: `Box ${boxData.labelIdentifier}`,
+            type: res.errors ? "error" : "success",
+            message: res.errors
+              ? "Error while trying to change number of items in the box"
+              : `Successfully removed ${boxFormValues?.numberOfItems} items from box`,
+          });
           onMinusClose();
         })
-        .catch((e) => {
-          // eslint-disable-next-line no-console
-          console.error("Error while trying to change number of items in the Box", e);
+        .catch(() => {
+          notificationVar({
+            title: `Box ${boxData.labelIdentifier}`,
+            type: "error",
+            message: "Error while trying to change number of items in the box",
```

Comment:
"Error: Could not add/remove items from the box." should be the error. Then what happens? Does the whole action fail? Should the user try again?

Reply:
@aerinsol the same here, the whole action fail, and use should try again.